efi/preinstall/errors.go

```diff
@@ -362,57 +361,47 @@ func (e *EmptyPCRBanksError) Error() string {
 	return fmt.Sprintf("the PCR %s missing from the TCG log but active and with one or more empty PCRs on the TPM", s)
 }
 
-// NoSuitablePCRAlgorithmError is returned wrapped in [TCGLogError] if there is no suitable PCR
-// bank where the log matches the TPM values when reconstructed. As multiple errors can occur
-// during testing (multiple banks and multiple PCRs), this error wraps each individual error
-// that occurred and provides access to them.
+// NoSuitablePCRAlgorithmError is returned wrapped in [TCGLogError] if it wasn't possible to
+// select a suitable PCR bank, which may happen under the following conditions:
+//   - The TCG log doesn't contain digests for a supported digest algorithm (SHA-256, SHA-384
+//     or SHA-512).
+//   - The TCG log is inconsistent with the TPM values when the log is reconstructed for one
+//     or more mandatory PCRs, for all algorithms in the log.
+//   - PCR 0 is mandatory and there is an issue with the way that the startup locality event
+//     (if present) is recorded.
+//   - There is a problem with the sequence of measurements that isn't specific to a PCR or
+//     a PCR bank, and which is incompatible with predicting PCR policies.
+//
+// As multiple errors can occur during testing, this error wraps each individual error that
+// occurred and provides access to them.
```

Comment:
I'm not sure whether it should be added  here or somewhere else but it doesn't seem to me that the doc comments explain the wrapping in *PCRError wrapper errors?

Reply:
I've added an extra bit to the documentation now, which I think is what you're looking for.